gamestonk_terminal/feature_flags.py

```diff
@@ -14,13 +14,10 @@
 USE_ION = strtobool(os.getenv("GTFF_USE_ION", "True"))
 
 # Enable Prompt Toolkit
-USE_PROMPT_TOOLKIT = strtobool(os.getenv("GTFF_USE_PROMPT_TOOLKIT", "False"))
+USE_PROMPT_TOOLKIT = strtobool(os.getenv("GTFF_USE_PROMPT_TOOLKIT", "True"))
```

Comment:
Prompt toolkit breaks graphs for me :(
With prompt toolkit graphs get stuck.
We thought this would be fixed with this https://github.com/GamestonkTerminal/GamestonkTerminal/blob/main/gamestonk_terminal/menu.py#L13, but latest dependencies are not working with this.

Reply:
I did not intend to push that flag.  Also interesting that my graphs are fine.

Reply:
> Prompt toolkit breaks graphs for me :(
> With prompt toolkit graphs get stuck.
> We thought this would be fixed with this https://github.com/GamestonkTerminal/GamestonkTerminal/blob/main/gamestonk_terminal/menu.py#L13, but latest dependencies are not working with this.

So is the default being False good enough or do I need to do something else to fix plots?